Core/clim-core/presentation-defs.lisp

```diff
@@ -1623,12 +1623,12 @@ protocol retrieving gestures from a provided string."))
 		(new-directory (cons
 				(first his-directory)
 				(append home (rest (rest his-directory))))))
-	     (make-pathname :host (pathname-host pathname)
-			    :device (pathname-device pathname)
-			    :name (pathname-name pathname)
-			    :version (pathname-version pathname)
-			    :type (pathname-type pathname)
-			    :directory new-directory))))
+	   (make-pathname :host (pathname-host pathname)
+			  :device (pathname-device pathname)
+			  :name (pathname-name pathname)
+			  :version (pathname-version pathname)
+			  :type (pathname-type pathname)
+			  :directory new-directory))))
```

Comment:
please detabify your changes. In emacs that would be:

(setq indent-tabs-mode nil)
M-x whitespace-cleanup